### DISCUSSION TITLE
ProHD Macintosh Revival Guide

### DISCUSSION THREAD

**Discussion by RandomMan719** [GerryDazoo/Slinger#314] [Show and tell]
Macintosh ProHD Revival Basic Guide (Custom Remote Guide below as well):

Thank you Mikoshi-nyudo 

Custom Remote for Mac Guide:
[How to use custom BIN files in SlingPlayer 1.0 for Mac.pdf](https://github.com/GerryDazoo/Slinger/files/10035788/How.to.use.custom.BIN.files.in.SlingPlayer.1.0.for.Mac.pdf)

I did this completely offline, just a router with ip address reservation for the slingbox, based off it's mac address. And a macbook pro hooked to the router. I have a 2010 mac with High Sierra OS.

Perform a hard reset, (doesn't seem to always take, maybe do it a couple times to make sure)
I then opened SlingPlayer 1.0.10.272
Instead of clicking on the slingbox, that it finds in the Directory, I clicked Add. Then you use direct IP, put in the ip address of the proHD, and port 5001. This creates an alias to the slingbox using IP address, not Finder ID.
Then edit the Alias you created.
And configure your sources and remote, change name and password. Then it shows some random error, doesn't even have a code.
The next screen says Completed Successfully. And that's it!

I ended up being able to scan for analog and digital channels on the internal tuner (It hangs on this step some times and you have to click Scan TV Channels twice usually, but if it hangs you can cancel and restart the process again). I also setup composite and component sources with custom remotes. It seems completely functional now. And I can use all configured sources and remotes with Slinger, and the password I created. During the setup process, if it asks for a password use "admin" but then you can change it during setup.

I also hooked it up to the Internet and bricked it again, and it took about 20 minutes to revive it again, counting rescanning for channels, which is very slow. I wanted to make sure the process was repeatable, and it was.  I am going to try it again with different mac, but same ProHD.

I also want to thank Mikoshi-nyudo for all the help!

![1](https://user-images.githubusercontent.com/116921195/202569974-a9bf10be-0242-4008-91fd-a5a3945074ce.jpg)
![2](https://user-images.githubusercontent.com/116921195/202570016-ab0c8ee2-4c94-4ffb-b46d-bdd5fd100a3b.jpg)
![3](https://user-images.githubusercontent.com/116921195/202570036-dbb611a9-2849-4207-8bf1-93d68d4ae2aa.JPG)
![4](https://user-images.githubusercontent.com/116921195/202570054-510af829-0297-4687-945c-ee1fe03f6306.jpg)
![5](https://user-images.githubusercontent.com/116921195/202570069-ee6717a7-ed56-40c3-8eb0-a2e118e672fc.JPG)
![6](https://user-images.githubusercontent.com/116921195/202570081-d68e76e6-64cd-4368-bbb9-287b7986b622.JPG)
![7](https://user-images.githubusercontent.com/116921195/202570116-3aba0144-9e7f-42c6-9ba7-72201cf71064.JPG)

**Comment by Mikoshi-nyudo** [comment]
Just to add a note: Mac OS dropped support for 32-bit applications and Carbon Framework support from Catalina onward. So you need to be running Mojave or earlier for the SingPlayer 1.0.10 app to work. Some people report issues with some old apps on Mojave. Neither @RandomMan719 or I have tested it on Mojave, so High Sierra or earlier is a sure bet..

**Reply by anonyada** [reply]
Problem: I wanted to run the older version of Slingplayer for MAC(1.0.10.dmg) on my High Sierra install.
You could install it...but Apples "Gatekeeper" on the later versions of MacOS (after Snow Leopard) would prevent you. Apple themselves, obviously black-listed the older application,
even though it had a valid cert. Asshats.
Therefore:
For anyone fighting this issue "Later versions of Apple OS will not let you run the older .DMG images after install IE: "Move it to Trash or Cancel..." Slingplayer.app versions......here you go:

From the CLI....sudo -i to root.

As root:1. root# ls -al@
This will show you all quarantined installed applications. Note: my Slingplayer.app was quarantined by Apple!
2. To remove the quarantine:  (Needs to be done recursively in the /Applications directory).
As root: root# xattr -cr Slingplayer.app
Reboot and test.

**Comment by kocoman2** [comment]
does this still work ?

**Reply by Mikoshi-nyudo** [reply]
Yes, based on the caveat of the older Mac OS requirement. It is also the better single-point solution to support the ATSC digital tuner support on Pro-HD units, unlike the Windows 1.5 PC version which requires some further work with ver 2.x to get the ProHD digital tuner to work for US NTSC boxes (ver 1.5 works with EU/UK PAL units and its DVB-T tuner).

**Reply by kocoman1** [reply]
can this work on 220/240 series? thx

**Reply by kocoman1** [reply]
so after setup in high sierra can use slinger by gerry ? how to block the slingbox from being online? filter thru openwrt/lede? thx, how to dl the old version of slingplayer, can run high sierra via osx-kvm ??

**Comment by anonyada** [comment]
Great write up. Thank you for sharing.
Do you have access to that original install .DMG for that specific Mac version of SlingPlayer 1.0.10.272?
I've looked everywhere for it and want to program my ATSC tuner.

**Reply by Mikoshi-nyudo** [reply]
You can find all the files you need for both Mac and PC under my post on HiFi remote, linked here;
http://www.hifi-remote.com/forums/viewtopic.php?p=145528#145528
